test/Router.t.sol

```diff
@@ -553,11 +694,35 @@ contract RouterTest is Test, LogicSignature, DelegationSignature {
         fees[0] = IParam.Fee(address(mockERC20), 0, bytes32(0));
         uint256 deadline = block.timestamp;
         IParam.LogicBatch memory logicBatch = IParam.LogicBatch(logics, fees, deadline);
-        bytes memory signature = getLogicBatchSignature(logicBatch, router.domainSeparator(), signerPrivateKey);
+        bytes memory signature = getTypedDataSignature(logicBatch, router.domainSeparator(), signerPrivateKey);
 
         vm.prank(delegatee);
         vm.expectRevert(IRouter.InvalidDelegatee.selector);
         vm.warp(expiry + 1);
         router.executeForWithSignerFee(user, logicBatch, signer, signature, tokensReturnEmpty, SIGNER_REFERRAL);
     }
+
+    function testInvalidateExecutionNonces() external {
+        uint256 newNonce = 10;
+        vm.expectEmit(true, true, true, true, address(router));
+        emit ExecutionNonceInvalidation(user, newNonce, 0);
+        vm.startPrank(user);
```

Comment:
```suggestion
        vm.prank(user);
```
seems we only need to be `user` for `router.invalidateExecutionNonces(newNonce)`. Suggest using `vm.prank`

Reply:
Please review all the `startPrank` as well. Thanks.